.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Fixes #ISSUE.
+
+
+## Pre-merge checklist
```

Comment:
## Experience in practice
I was just doing #432 and I decided to try the new template. I guess I do prefer the checkbox changes I suggested, e.g. 1 checkbox for each section instead of 2. But I'd be willing to consider changing it back as well. Because it is now less clear which is true. Did I check off "Docs" because I added docs? Or did I check it off because no docs had to be added?

Reply:
Action item?

Reply:
For "High-level description", I made a [new suggestion](https://github.com/monarch-initiative/mondo-ingest/pull/430#discussion_r1482031772).

For "Checkboxes", the action item here is that I want you or @twhetzel to take what I've written in consideration and decide again which is better: (a) 1 list of checkboxes where we know that the checklist is done when 100% of the boxes are checked, or (b) go back to sections (Docs, Review, etc) for each part of the checklist, and consider the checklist done if at least 1 box in each section is checked.

Reply:
I like the way it is now!